new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -83,6 +84,8 @@
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 @DisplayName("Test to create model in image domain and start the domain")
 @ExtendWith(Timing.class)
+@ExtendWith(IntegrationTestWatcher.class)
```

Comment:
maybe we should just have a single extension - are there example of when we would would want one of these but not the other @ddsharpe @rjeberhard? i can't think of any

Reply:
Removed the Timing extension and moved the @ExtendWith(IntegrationTestWatcher.class)  from test class to IntegrationTest interface.

The IntegrationTestWatcher implements methods that were in Timing.java